### PR TITLE
AGENT-275: Add new agent graph command to output agent internal dependency graph

### PR DESCRIPTION
**Pull request by andfasano**
This patch adds the equivalent installer `graph`  command for the `agent` specific targets

### CODE REVIEW

File: cmd/openshift-install/graph.go [hunk 1]
```diff
@@ -42,7 +44,7 @@ func runGraphCmd(cmd *cobra.Command, args []string) error {
 		string(gographviz.Shape): "box",
 		string(gographviz.Style): "filled",
 	}
-	for _, t := range targets {
+	for _, t := range cmdTargets {
```
Comment (jhixson74) [inline cmd/openshift-install/graph.go:47]: What happens to targets? This appears to throw it out.

Comment (zaneb) [reply cmd/openshift-install/graph.go:47]: It's now passed as an argument on l. 30.

**Review by zaneb — COMMENTED** [review]
/lgtm

It might be worth also generating the graph and adding it to the docs like the main one is in https://github.com/openshift/installer/blob/master/docs/design/assetgeneration.md#dependency-graph

**Pull request merged**